types/p2p.yaml

```diff
@@ -27,19 +24,14 @@ MetaData:
   required: ["seq_number", "attnets"]
   properties:
     seq_number:
-      allOf:
-        - description: "Uint64 starting at 0 used to version the node's metadata. If any other field in the local MetaData changes, the node MUST increment seq_number by 1."
-        - $ref: "./primitive.yaml#/Uint64"
+      $ref: "./primitive.yaml#/Uint64"
+      description: "Uint64 starting at 0 used to version the node's metadata. If any other field in the local MetaData changes, the node MUST increment seq_number by 1."
     attnets:
-      allOf:
-        - description: "Bitvector representing the node's persistent attestation subnet subscriptions."
-        - $ref: "./primitive.yaml#/Hex"
-        - example: "0x0000000000000000"
```

Comment:
Need to be careful when removing examples, the default `Hex` example does not use the length you would expect for `attnets` whereas the override example does